src/subsplit_dag.cpp

```diff
@@ -124,14 +125,14 @@ StringSizeMap SubsplitDAG::SummaryStatistics() const {
 }
 
 void SubsplitDAG::Print() const {
-  for (const auto &dag_node : dag_nodes_) {
-    std::cout << dag_node->ToString() << std::endl;
+  for (auto dag_node : storage_.GetVertices()) {
+    std::cout << dag_node.ToString() << std::endl;
   }
 }
 
 void SubsplitDAG::PrintNodes() const {
-  for (const auto &dag_node : dag_nodes_) {
-    std::cout << dag_node->Id() << ": " << dag_node->GetBitset().SubsplitToString()
+  for (const auto &dag_node : storage_.GetVertices()) {
+    std::cout << dag_node.Id() << ": " << dag_node.GetBitset().SubsplitToString()
               << std::endl;
   }
 }
```

Comment:
Is there a reason `dag_node`'s `const` ref was dropped in one function but not the other?

Reply:
I'd like to drop it everywhere, because SubsplitDAGNode now is a lightweight object, meant to be passed by value like an iterator or string_view.